e2e/src/utils/retries.js

```diff
@@ -24,7 +25,7 @@ export const launchApp = async (
   })
 }
 
-export const reloadReactNative = async () => {
+export const reloadReactNative = async (dismissBottomSheet = true) => {
```

Comment:
whether to dismiss the bottom sheet seems like a weird dependency for a RN reload function to have? is there anywhere where this is false? if not then should we just remove this function input?

aside: does it make sense to add the dismiss bottom sheet action to somewhere like `quickOnboarding`?

Reply:
`dismissBottomSheet` is present in the quickOnboarding [here](https://github.com/valora-inc/wallet/pull/3049/files#diff-6fde14f432e5ec2dba3efc8a02ff8a389c62db2d6753bc2987d6064a218ce097).

Will take a look at removing the dismissBottomSheet. It would be great to avoid waiting 30 seconds on `reloadReactNative` if the bottom sheet isn't present.

Reply:
Side note, I feel we should stop using `<Modal />` as they conflict with the normal navigation. And only a single one can be displayed at the same time. Causing issue with shake to report a bug for instance.
But may be too much work now 😅 .